### DISCUSSION TITLE
Editable E-Mails

### DISCUSSION THREAD

**Discussion by Pankraz01** [pterodactyl/panel#2928] [Feature Requests]
There is a possibility to connect your e-mail server to the Panel. This is how the users will be notified. What I miss, there are no templates that you can edit. Because unfortunately there is only one standard mail and it is not modifiable. Maybe you can come up with an idea using RTF documents or RTF templates. A simple text column would of course also be ideal for the beginning.

happy new year

**Comment by matthewpi** [comment]
https://github.com/pterodactyl/panel/tree/develop/resources/views/vendor/notifications

**Reply by Pankraz01** [reply]
thanks for that. is there any documentation to read, to get through this?

**Comment by FoksVHox** [comment]
As @matthewpi pointed out, you're able to edit the emails. However, I see your point with the being it isn't that user-friendly, the solution that I see is that the panel gets an email editor, which would be far to much work, for what it is worth.
